djes/management/commands/sync_es.py

```diff
@@ -35,10 +35,14 @@ def get_indexes():
 def get_latest_index_version(name):
     conn = connections.get_connection('default')
     alias = conn.indices.get_alias(name)
+    if not alias:
```

Comment:
non-existent alias raised an exception in Travis:  

`NotFoundError: TransportError(404, u'IndexMissingException[[avclub-integration] missing]')`

https://magnum.travis-ci.com/theonion/avclub/builds/16386430